### PR TITLE
Red Hat Mono: Version 1.010 added

### PR DESCRIPTION
**Pull request by RosaWagner**
f9e0fb5: [gftools-packager] Red Hat Mono: Version 1.010 added

* Red Hat Mono Version 1.010 taken from the upstream repo https://github.com/jeremymickel/RedHatFonts at commit https://github.com/jeremymickel/RedHatFonts/commit/9dd2dd530e8dc986418f56d1e11c9c246bfe2bc7.

 7187819: [gftools-packager] ofl/redhatmono remove METADATA "source".  google/fonts#2587

### CODE REVIEW

**Comment by RosaWagner** [comment]
Updated

## Red Hat Mono: Version 1.010 added

---
 6ec2f30: [gftools-packager] Red Hat Mono: Version 1.010 added

* Red Hat Mono Version 1.010 taken from the upstream repo https://github.com/jeremymickel/RedHatFonts at commit https://github.com/jeremymickel/RedHatFonts/commit/9dd2dd530e8dc986418f56d1e11c9c246bfe2bc7.

 209eddf: [gftools-packager] ofl/redhatmono remove METADATA "source".  google/fonts#2587

**Comment by RosaWagner** [comment]
1. The instances in the fvar table are not in the right order, but after testing in several app, it doesn't mess up the dropdown menu. I do think that it should be fixed though, because we don't know how non-tested environments would react.

2. The missing SemiBold instance has been discussing with the designer during a meeting. He is against including it. I leave @davelab6 decides if he wants to merge it anyway given that this font was prioritised, or continuing the conversation with the designer to convince him otherwise.

File: ofl/redhatmono/OFL.txt [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Copyright 2018 The Fraunces Project Authors (https://github.com/undercasetype/Fraunces)
```
Comment (davelab6) [inline ofl/redhatmono/OFL.txt:1]: This should be fixed

Comment (RosaWagner) [reply ofl/redhatmono/OFL.txt]: I linked the OFL directly with packager, it is the actual file upstream. How could I miss it ?!

Comment (RosaWagner) [reply ofl/redhatmono/OFL.txt]: I made a PR upstream

**Comment by m4rc1e** [comment]
LGTM. Thanks

**Pull request merged**